pkgs/by-name/qq/qqsp/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  adwaita-qt,
+  libadwaita,
+  adw-gtk3,
+  gst_all_1,
+  gnome-themes-extra,
+  librsvg,
+  qt5,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "qqsp";
+  version = "1.9";
+  src = fetchFromGitHub {
+    owner = "Sonnix1";
+    repo = "Qqsp";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-eDgoa+/dcJ8Ti+YLHgKUKus0+zRrFEuJ19wUpbFpcBU=";
+  };
+  buildInputs = [
+    adwaita-qt
+    libadwaita
+    adw-gtk3
+    gnome-themes-extra
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+    qt5.qtmultimedia
+    qt5.qtwebengine
+    qt5.qtbase
+  ];
+  nativeBuildInputs = [
+    qt5.qmake
+    qt5.qtbase
+    qt5.qtmultimedia
+    qt5.qtwebengine
+    qt5.wrapQtAppsHook
+    librsvg
+  ];
```

Suppress the new GCC 14 error
(the reason why you can not see the error is because your branch is old)
```suggestion
  ];
  env.NIX_CFLAGS_COMPILE = "-Wno-incompatible-pointer-types";
```

---

I would not have found this without @FliegendeWurst finding it first and leaving that really helpful comment https://github.com/NixOS/nixpkgs/pull/363124#issuecomment-2614511760
I hope my suggested change is acceptable to you, if you would rather see a patch I could give it a try, just let me know :)
However the last commit to the project is around 6 years old, so the patch would not get upstreamed